spec/models/notification_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe Notification do
+  it { should belong_to(:user) }
+  it { should belong_to(:topic) }
+
+  describe '#mark_as_read' do
+    it 'sets read status to true' do
+      notification = build(:notification, read: false)
+
+      notification.mark_as_read
+
+      expect(notification.read).to be_truthy
```

Comment:
There is a shortcut for methods that reutnr a boolean like `notification.read`

In specs, you can directly assert : `expect(notification).to be_read` ;)